### ISSUE TITLE
[Bug]: Way too many Florists in every village

### ISSUE THREAD

**Issue by Alexsys** [samedifferent/morevillagers#75]
### Mod Version

latest of 1.18.2

### Mod Loader

Forge

### Mod Loader Version

latest of 1.18.2

### What happened?

I love your mod, its such a delight

I noticed there are way too many florists in every village, no other jobs spawn like butcher

75% are florists every time

i love u

### latest.log

wat log

**Comment by ncottrill** [comment]
I agree. Too many of all of the More Villagers professions. Each village just needs one. The thing is, More Villagers adds a bunch of teeny buildings - some too small for a bed - to villages. This wrecks the aesthetics of any mod that builds beautiful villages like Better Villages or Repurposed Structures. I'd go so far as to say "Don't add any buildings" ... just add the profession blocks that players can craft later.

**Comment by broadcast** [comment]
I have this problem as well. There's a config to tweak the weight of the house chance generation.

@ncottrill You can disable all house generation in the config for this mod to only get the profession blocks.
